pss/pss_test.go

```diff
@@ -614,8 +614,8 @@ func TestOutboxFull(t *testing.T) {
 	if err == nil {
 		t.Fatalf("expected error enqueing third message, instead got nil")
 	}
-	procChan<- struct{}{}
-	procChan<- struct{}{}
+	procChan <- struct{}{}
+	procChan <- struct{}{}
 	//Must wait a bit for the routines processing the messages to free the slots
 	time.Sleep(1 * time.Millisecond)
```

Comment:
I really don't like sleeps in tests.

Reply:
Me neither, but is the only way I've found to be sure the processing routine finish freeing the slot. Maybe I should loop with timeout until that.

Reply:
Sleep removed. Instead used a timed waiting in the slots channel
```go
//There should be a slot again in the outbox
select {
case <-ps.outbox.slots:
case <-time.After(2 * time.Second):
    t.Fatalf("timeout waiting for a free slot")
}
```